chevah/compat/tests/normal/test_capabilities.py

```diff
@@ -68,13 +77,16 @@ def test_impersonate_local_account_windows(self):
         administration mode and disabled for the rest.
 
         See: runningAsAdministrator
+        See: hasUAC
```

Comment:
not sure what to say... we are using this method only in a single test... maybe we should make it an embedded helper method to make it closer to the test code and so that we don't need this `See: hasUAC` docststring... this docstring feels strage